### ISSUE TITLE
i.sentinel.import: wrong resolution set for latlong locations

### ISSUE THREAD

**Issue by veroandreo** [OSGeo/grass-addons#84]
**Name of the addon**
i.sentinel.import

**Describe the bug**
When trying to import S2 scenes into a latlong location using `i.sentinel.import` with -r flag, the resolution is set to 10, 20 or 60 degrees instead of the equivalent of 10, 20 and 60 m in degrees. If I import into a UTM location all is fine, as well as if I use `r.import`instead of i.sentinel.import with a latlong location. 
As you can see below, r.import estimates a resolution, however, r.sentinel.import uses the nominal resolution in meters anyway.

**To Reproduce**
1. Download a S2 scene
```
i.sentinel.download settings=your_sentinel_file output=tmp uuid=541021f8-63f5-4e9d-ba28-425c4c8451df
```
2. Import into a latlong location - here, I already got messages that the imported map was empty
```
i.sentinel.import input=tmp/ pattern='B(02_1|03_1|04_1|08_1|8A_2)0m' extent=input memory=500
...
Estimated target resolution for input band
<T21JYM_20200126T134201_B11_20m>: 0.00018606630466763683
Using given resolution for input band <T21JYM_20200126T134201_B11_20m>:
20.0
Reprojecting <T21JYM_20200126T134201_B11_20m>...
ERROR: The reprojected raster <T21JYM_20200126T134201_B11_20m> is empty
Writing metadata to maps...
```
3. Run r.info to check (see below)

**Screenshots**
![image](https://user-images.githubusercontent.com/20075188/73405459-3e209e80-42f4-11ea-8def-296c3a4951c1.png)

**System description (please complete the following information):**
 - Operating System: Linux Fedora 31
 - GRASS GIS version 7.9 (dev)
 - with python 3.7.4 because of ctypes problem

**Comment by neteler** [comment]
Probably we need a change here for LongLat:

https://github.com/OSGeo/grass-addons/blob/f5b90b741b09a63269ee8e782ebcc43378018a08/grass7/imagery/i.sentinel/i.sentinel.import/i.sentinel.import.py#L229

**Comment by veroandreo** [comment]
ping @landam  :) 
any chance to have a look at this?

**Comment by lucadelu** [comment]
> Probably we need a change here for LongLat:
> 
> https://github.com/OSGeo/grass-addons/blob/f5b90b741b09a63269ee8e782ebcc43378018a08/grass7/imagery/i.sentinel/i.sentinel.import/i.sentinel.import.py#L229

The change should be done in [_raster_resolution function](https://github.com/OSGeo/grass-addons/blob/f5b90b741b09a63269ee8e782ebcc43378018a08/grass7/imagery/i.sentinel/i.sentinel.import/i.sentinel.import.py#L258), the problem it is to get the right value in degree according to the position on the Earth surface.

Could [geodesic lenght](https://pyproj4.github.io/pyproj/stable/examples.html#geodesic-line-length) be useful?

**Comment by neteler** [comment]
> ... the problem it is to get the right value in degree according to the position on the Earth surface.
> 
> Could [geodesic lenght](https://pyproj4.github.io/pyproj/stable/examples.html#geodesic-line-length) be useful?

We have `length_geodesic()`, maybe interesting?

- doc: https://grass.osgeo.org/grass79/manuals/libpython/pygrass.vector.html?highlight=length_geodesic#pygrass.vector.geometry.Line.length_geodesic
- code: https://github.com/OSGeo/grass/blob/90634b76e3bcf9695a844cf3f6581975fa7283d9/lib/python/pygrass/vector/geometry.py#L848

**Comment by veroandreo** [comment]
In [this commit](https://github.com/veroandreo/i.landsat/commit/1b562764a46cbf9b863950f7e68383bffb014086) in the [i.landsat.import](https://github.com/veroandreo/i.landsat) addon I propose a simple approach, i.e., using r.import's `resolution = estimated` when the target location is longlat. What do you think @neteler and @lucadelu ?

**Comment by lucadelu** [comment]
It could work, did you try if you addons works properly?

**Comment by veroandreo** [comment]
> It could work, did you try if you addons works properly?

Yes, It works as expected. It's `r.import` who does the resolution estimation for latlong locations.
